<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtM1PlayerDemoClass</class>
 <widget class="QMainWindow" name="QtM1PlayerDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtM1PlayerDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1,0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QProgressBar" name="pbProgress">
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="tvTime">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="btPlay">
          <property name="text">
           <string>停止播放</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btAudio">
          <property name="text">
           <string>停止音频</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btSelectFile">
          <property name="text">
           <string>选择文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btInput">
          <property name="text">
           <string>输入url</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btReverse">
          <property name="text">
           <string>翻转</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btScreenShot">
          <property name="text">
           <string>截图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>解码:</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="llViewMode">
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>模式:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="tvInfo">
        <property name="text">
         <string>媒体信息</string>
        </property>
        <property name="margin">
         <number>8</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
